Tree_Inorder_Traversal/Inorder_Traversal.kt

```diff
@@ -0,0 +1,56 @@
+class Node
+{
+    int num;
+    Node left, right; //Structure of Binary tree node
+
+    public node(int num)
+    {
+        key = item;
+        left = right = null;
+    }
+}
+
+class BinaryTree
+{
+    Node root;
+
+    BinaryTree()
+    {
+        root = null;
+    }
+
+    fun printInorder(Node: node):void	//function to print inorder traversal	
```

Comment:
Please remove the comments from the sides and add on a separate line before the function.